### ISSUE TITLE
Modifier keys for "Enable popup with original text"

### ISSUE THREAD

**Issue by Dook1** [translate-tools/linguist#408]
Would you please add the option to select modifier keys(Ctrl, Shift, Alt) for "Enable popup with original text"?

**Comment by vitonsky** [comment]
Yes, i can implement this feature if it would be useful for users. Could you please describe your use case, why you need this feature and how you imagine usage of this feature?

Your description are important to decide should this feature be implemented or not, so please try to be detailed, try to tell why you want this behavior.

**Comment by Dook1** [comment]
Constant pop-ups bother me because they interfere with reading and I have to always keep the mouse away from the text.  I only need to look at the original on rare occasions:
 1) when the translation is not clear enough.
 2) when the translator translated a word that is a term
 3) when the word is translated without taking into account the context

**Comment by vitonsky** [comment]
I see your concern about popup with original text for full-page translation, that appears by hover on text feels bad for some cases.

What solution you see? How exactly it should work?

**Comment by vitonsky** [comment]
Good idea. I think it must be implemented

**Comment by Dook1** [comment]
Thanks
